css/properties.json

```diff
@@ -3332,17 +3332,18 @@
     "status": "standard"
   },
   "column-gap": {
-    "syntax": "<length-percentage> | normal",
+    "syntax": "normal | <length-percentage>",
     "media": "visual",
     "inherited": false,
     "animationType": "lpc",
-    "percentages": "referToContentWidthOfMulticolElement",
+    "percentages": "referToDimensionOfContentArea",
     "groups": [
-      "CSS Columns"
+      "CSS Columns",
+      "CSS Box Alignment"
```

Comment:
This group is not defined yet. I guess that's will be fixed when #185 will be merged.

Reply:
That's correct! Btw. how long it usually takes to get PR merged in?

Reply:
Yes, and it's already merged.